src/EnsembleKalmanProcess.jl

```diff
@@ -448,20 +457,109 @@ function get_cov_blocks(cov::AbstractMatrix{FT}, p::IT) where {FT <: Real, IT <:
     return uu_cov, ug_cov, gg_cov
 end
 
+"""
+    multiplicative_inflation!(
+        ekp::EnsembleKalmanProcess;
+        s,
+    ) where {FT, IT}
+Applies multiplicative noise to particles.
+Inputs:
+    - ekp :: The EnsembleKalmanProcess to update.
+    - s :: Scaling factor for time step in multiplicative perturbation.
+"""
+function multiplicative_inflation!(ekp::EnsembleKalmanProcess; s::FT = 1.0) where {FT <: Real}
+
+    scaled_Δt = s * ekp.Δt[end]
+
+    if scaled_Δt >= 1.0
+        error(string("Scaled time step: ", scaled_Δt, " is >= 1.0", "\nChange s or EK time step."))
+    end
+
+    u = get_u_final(ekp)
+    u_mean = get_u_mean_final(ekp)
+    prefactor = sqrt(1 / (1 - scaled_Δt))
+    u_updated = u_mean .+ prefactor * (u .- u_mean)
+    ekp.u[end] = DataContainer(u_updated, data_are_columns = true)
+
+end
+
+"""
+    additive_inflation!(
+        ekp::EnsembleKalmanProcess;
+        prior::Union{ParameterDistribution, Nothing} = nothing,
+        s::FT = 1.0,
+    ) where {FT <: Real}
+Applies additive gaussian noise to particles. Noise is drawn from normal distribution with 0 mean
+and scaled parameter covariance. If no prior provided (default), scales parameter covariance matrix from
+current ekp iteration. If prior is provided, the parameter covariance is calculated from
+the prior distribution.
+Inputs:
+    - ekp :: The EnsembleKalmanProcess to update.
+    - s :: Scaling factor for time step in additive perturbation.
+    - prior :: Prior parameter distribution used for additive inflation parameter covariance calculation.
+        If prior=nothing, the parameter covariance from the current iteration is used.
+"""
+function additive_inflation!(
+    ekp::EnsembleKalmanProcess;
+    prior::Union{ParameterDistribution, Nothing} = nothing,
+    s::FT = 1.0,
+) where {FT <: Real}
+
+    scaled_Δt = s * ekp.Δt[end]
+
+    if scaled_Δt >= 1.0
+        error(string("Scaled time step: ", scaled_Δt, " is >= 1.0", "\nChange s or EK time step."))
+    end
+
+    if isnothing(prior)
+        Σ = get_u_cov_final(ekp)
+    else
+        Σ = cov(prior)
+    end
```

Comment:
Σ = isnothing(prior) ? get_u_cov_final(ekp) : cov(prior)

Reply:
To add to this, the prior is already stored in EKP, so 

So we could just pass in a flag, for example 
`use_prior_cov::Bool = false` 
and then here:
`Σ = use_prior_covariance ?  cov(get_u_prior(ekp)) : get_u_cov_final(ekp)`

I don't think we envisage other inflation distributions here, so I presume we do not need the ability to pass a generic one in

Reply:
As far as I can tell `get_u_prior(ekp)` will get the initial parameter ensemble that is randomly drawn from the prior. So calling `cov(get_u_prior(ekp))` will give an approximation of the prior covariance; whereas calling `cov(prior)` (passing the prior distribution) will give the true prior covariance.  No sure if it makes a practical difference, but that's the only reason I went with passing the prior ParameterDistribution, but happy to switch to `cov(get_u_prior(ekp))` if that makes more sense.

Reply:
This is a good point, but using the ensemble to estimate the prior cov is still consistent with the default option (to use the current ensemble to estimate the current covariance) so I doubt there will be loss of accuracy here.